x/group/client/testutil/tx.go

```diff
@@ -629,10 +594,10 @@ func (s *IntegrationTestSuite) TestTxUpdateGroupMembers() {
 			append(
 				[]string{
 					val.Address.String(),
-					"2",
```

Comment:
I'm a bit wary of changing other tests. Is there a reason we need to do this?

Reply:
The test ordering was messing up the IDs for groups